components/brave_rewards/resources/ui/components/settingsPage.tsx

```diff
@@ -41,6 +41,7 @@ class SettingsPage extends React.Component<Props, {}> {
     this.actions.getConfirmationsHistory()
     this.actions.getExcludedPublishersNumber()
     this.actions.getAdsData()
+    this.actions.getAdsIsSupportedRegion()
```

Comment:
This can be moved into `componentDidMount` as this will not change and don't need to be in refresh function

Reply:
done